packages/leancode_analytics/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: leancode_analytics
 description: A new Flutter package project.
-version: 0.0.1
+version: 0.0.2
```

Comment:
Its not published so i dont think you need to worry about it

Reply:
Yeah, but @KamilSztandur already using it in lncd project